pyproject.toml

```diff
@@ -52,6 +52,7 @@ pydantic-settings = { version = "^2.5.2", optional = true }
 sqlalchemy = { version = "^2.0.35", optional = true }
 sqlalchemy-utils = { version = "^0.41.2", optional = true }
 fastapi = { version = "^0.115", optional = true}
+asgi-correlation-id = {version = ">=4.2.0", optional = true, python = ">=3.8"}
```

Comment:
BTW, why adding python contraing while python version is already fixed `python = "^3.11"`